io_scene_nif/properties/object.py

```diff
@@ -70,127 +64,101 @@ def __init__(self):
-            type=cls,
-        )
-
-        cls.rootnode = EnumProperty(
-            name='Nif Root Node',
-            description='Type of property used to display meshes.',
-            items=(
-                ('NiNode', 'NiNode', "", 0),
-                ('BSFadeNode', 'BSFadeNode', "", 1)),
-            default='NiNode',
-        )
-
-        cls.prn_location = EnumProperty(
-            name='Weapon Location',
-            description='Attachment point of weapon, for Skyrim, FO3 & Oblivion',
-            items=[(item, item, "", i) for i, item in enumerate(["NONE","BACK","SIDE","QUIVER","SHIELD","HELM","RING"])],
-            # default = 'NONE'
-        )
-
-        cls.bsnumuvset = IntProperty(
-            name='BS Num UV Set',
-            default=0
-        )
-
-        cls.longname = StringProperty(
-            name='Nif Long Name'
-        )
-
-        cls.consistency_flags = EnumProperty(
-            name='Consistency Flag',
-            description='Controls animation type',
-            items=[(item, item, "", i) for i, item in enumerate(NifFormat.ConsistencyType._enumkeys)],
-            # default = 'SHADER_DEFAULT'
-        )
-
-        cls.objectflags = IntProperty(
-            name='Object Flag',
-            description='Controls animation and collision',
-            default=0
-        )
-
-        cls.bsxflags = IntProperty(
-            name='BSX Flags',
-            description='Controls animation and collision',
-            default=0  # 2 = Bit 1, enable collision
-        )
-
-        cls.upb = StringProperty(
-            name='UPB',
-            description='Commands for an optimizer?',
-            default=''
-        )
-
-        cls.extra_data_store = PointerProperty(
-            name="Extra Data",
-            description="Used to store all the Extra data",
-            type=ExtraDataStore,
-        )
-
-    @classmethod
-    def unregister(cls):
-        del bpy.types.Object.niftools
+
+    rootnode: EnumProperty(
+        name='Nif Root Node',
+        description='Type of property used to display meshes.',
+        items=(
+            ('NiNode', 'NiNode', "", 0),
+            ('BSFadeNode', 'BSFadeNode', "", 1)),
+        default='NiNode',
+    )
+
+    prn_location: EnumProperty(
+        name='Weapon Location',
+        description='Attachment point of weapon, for Skyrim, FO3 & Oblivion',
+        items=[(item, item, "", i) for i, item in enumerate(["NONE","BACK","SIDE","QUIVER","SHIELD","HELM","RING"])],
+        # default = 'NONE'
+    )
+
+    bsnumuvset: IntProperty(
+        name='BS Num UV Set',
+        default=0
+    )
+
+    longname: StringProperty(
+        name='Nif Long Name'
+    )
+
+    consistency_flags: EnumProperty(
+        name='Consistency Flag',
+        description='Controls animation type',
+        items=[(item, item, "", i) for i, item in enumerate(NifFormat.ConsistencyType._enumkeys)],
+        # default = 'SHADER_DEFAULT'
+    )
+
+    objectflags: IntProperty(
```

Comment:
Should we go with "flags" to be consistent of usage.

Reply:
Done